packages/plugin-lighthouse/src/lib/lighthouse-plugin.integration.test.ts

```diff
@@ -1,12 +1,57 @@
-import { expect } from 'vitest';
-import { pluginConfigSchema } from '@code-pushup/models';
-import { lighthousePlugin } from './lighthouse-plugin';
+import { rm } from 'node:fs/promises';
+import { join } from 'node:path';
+import { afterEach, expect } from 'vitest';
+import { AuditOutput, pluginConfigSchema } from '@code-pushup/models';
+import { getLogMessages } from '@code-pushup/test-utils';
+import { ui } from '@code-pushup/utils';
+import { getRunner, lighthousePlugin } from './lighthouse-plugin';
 
 describe('lighthousePlugin', () => {
   it('should create valid plugin config', () => {
-    const pluginConfig = lighthousePlugin('https://code-pushup-portal.com');
+    const pluginConfig = lighthousePlugin('https://www.google.com/');
     expect(() => pluginConfigSchema.parse(pluginConfig)).not.toThrow();
     expect(pluginConfig.audits.length).toBeGreaterThan(100);
     expect(pluginConfig.groups).toHaveLength(5);
   });
 });
+
+describe('getRunner', () => {
+  const getRunnerTestFolder = join('tmp', 'plugin-lighthouse', 'get-runner');
+
+  afterEach(async () => {
+    await rm(getRunnerTestFolder, { recursive: true, force: true });
+  });
```

Comment:
🧪 Test outputs : I noticed one more thing - the whole `plugin-lighthouse` folder should be removed after the tests.

I recommend adding an `afterAll` hook for this.
```ts
 afterAll(async () => {
    await rm(join('tmp', 'plugin-lighthouse'), { recursive: true, force: true });
  });
```

Reply:
added